tests/run-pass/ptr_offset.rs

```diff
@@ -60,7 +60,7 @@ fn ptr_offset() {
     // fn ptr offset
     unsafe {
         let p = f as fn() -> i32 as usize;
-        let x = (p as *mut u32).offset(0) as usize;
+        let x = (p as *mut u32).offset(0);
```

Comment:
I think the intent of the test is to `offset` the original value, so probably this should be a `transmute` to `*mut u32`.